secure-app/backend/src/db/init.sql

```diff
@@ -12,34 +12,82 @@ CREATE TABLE IF NOT EXISTS Users (
 
 CREATE TABLE IF NOT EXISTS Editeurs (
     idEditeur SERIAL PRIMARY KEY,
-    nomEditeur TEXT NOT NULL
+    libelleEditeur TEXT NOT NULL,
+    exposant BOOLEAN TEXT NOT NULL,
+    distributeur BOOLEAN TEXT NOT NULL DEFAULT FALSE,
+    logoEditeur TEXT NOT NULL
 );
 
+CREATE TABLE IF NOT EXISTS Reservant {
+    idReservant SERIAL PRIMARY KEY, 
+    idEditeur INTEGER REFERENCES Editeurs(idEditeur),
+    libelleReservant TEXT NOT NULL, 
+    typeReservant TEXT CHECK (typeReservant IN ('editeur','autre'))
+}
+
```

Comment:
Here I'd be better to keep one Reservant table, delete the Editeurs' one and verify for each type of reservant

```suggestion
CREATE TABLE IF NOT EXISTS Reservant {
    idReservant SERIAL PRIMARY KEY,
    libelleReservant TEXT NOT NULL,
    estExposant BOOLEAN NOT NULL,
    estDistributeur BOOLEAN  NOT NULL DEFAULT FALSE,
    estPrestataire BOOLEAN NOT NULL DEFAULT FALSE,
    estEditeur BOOLEAN NOT NULL DEFAULT TRUE,
    estBoutique BOOLEAN NOT NULL DEFAULT FALSE,
    estAssociation BOOLEAN NOT NULL DEFAULT FALSE,
    logoReservant TEXT NOT NULL
}
```